### PR TITLE
Update/prune deps with `go mod tidy` and `gazelle -- update-repos`

### PR DESCRIPTION
**Pull request by tempoz**
Ran the enterprise server after this change, encountered no dependency problems at all.

---

**Version bump**: Patch

### CODE REVIEW

**Review by tylerwilliams — APPROVED** [review]
LGTM

**Comment by tempoz** [comment]
deps.bzl was not run with -prune, that will need to wait until after the internal repo is merged.

**Pull request merged**
